azure-devops/enforce-clang-format.cmd

```diff
@@ -0,0 +1,6 @@
+@echo off
+:: Copyright (c) Microsoft Corporation.

```

Comment:
The banner should go at the very top whenever possible; in this case, it is totally fine for it to occur before the `@echo off`. (The product sources have a special first line, so they don't follow this.) For clarity, I recommend a newline after.

Reply:
If they go before `@echo off` I think they get spewed in the output?

Reply:
Just tested - they don't. At least not with a plain old batch file.

Reply:
Apparently not. Fixed!